### PR TITLE
Bpk 4313 migrate apps docs pages: A-B

### PR DESCRIPTION
**Pull request by olliecurtis**
This PR migrates the app pages from components A-B.

Changes:
- Fix for an issue with `Text` app page not loading images correctly
- Moved components A to B app pages to `.mdx`
- Updated import paths to new format
- Moved Web barchart to separate file for cleanup

### CODE REVIEW

File: docs/src/pages/components/Text/AndroidText.mdx [hunk 1]
```diff
@@ -11,27 +11,35 @@ import AndroidComponentScreenshots from '../../../components/Screenshots/Android
 import textReadme from '../../../../../backpack-android/docs/Text/README.md';
 import textSpanReadme from '../../../../../backpack-android/docs/TextSpans/README.md';
 
+import TextDefault from '../../../../../backpack-android/docs/Text/screenshots/default.png';
+import TextDefaultDm from '../../../../../backpack-android/docs/Text/screenshots/default_dm.png'
+import TextEmphasized from '../../../../../backpack-android/docs/Text/screenshots/emphasized.png'
+import TextEmphasizedDm from '../../../../../backpack-android/docs/Text/screenshots/emphasized_dm.png'
+import TextHeavy from '../../../../../backpack-android/docs/Text/screenshots/heavy.png'
+import TextHeavyDm from '../../../../../backpack-android/docs/Text/screenshots/heavy_dm.png'
```
Comment (georgegillams) [inline docs/src/pages/components/Text/AndroidText.mdx:19]:
```suggestion
import TextDefault from 'backpack-android/docs/Text/screenshots/default.png';
import TextDefaultDm from 'backpack-android/docs/Text/screenshots/default_dm.png'
import TextEmphasized from 'backpack-android/docs/Text/screenshots/emphasized.png'
import TextEmphasizedDm from 'backpack-android/docs/Text/screenshots/emphasized_dm.png'
import TextHeavy from 'backpack-android/docs/Text/screenshots/heavy.png'
import TextHeavyDm from 'backpack-android/docs/Text/screenshots/heavy_dm.png'
```

File: docs/src/pages/components/Text/NativeText.mdx [hunk 2]
```diff
@@ -9,30 +9,37 @@ import NativeComponentScreenshots from '../../../components/Screenshots/NativeCo
 
 import readme from '../../../../../backpack-react-native/lib/bpk-component-text/README.md';
 
+import iosTextDefault from '../../../../../backpack-react-native/lib/bpk-component-text/screenshots/ios/default.png';
+import androidTextDefault from '../../../../../backpack-react-native/lib/bpk-component-text/screenshots/android/default.png';
+import iosTextEmphasized from '../../../../../backpack-react-native/lib/bpk-component-text/screenshots/ios/emphasize.png';
+import androidTextEmphasized from '../../../../../backpack-react-native/lib/bpk-component-text/screenshots/android/emphasize.png'
+import iosTextHeavy from '../../../../../backpack-react-native/lib/bpk-component-text/screenshots/ios/heavy.png';
+import androidTextHeavy from '../../../../../backpack-react-native/lib/bpk-component-text/screenshots/android/heavy.png'
```
Comment (georgegillams) [inline docs/src/pages/components/Text/NativeText.mdx:17]: same

File: docs/src/pages/components/Text/AndroidText.mdx (hunk 1 above)
Comment (georgegillams) [reply docs/src/pages/components/Text/AndroidText.mdx:19]: tbh I think it's better to just do this inline. Makes it easier to see what's what, and means there's less noise in PR MD previews (as they just omit the components).

Comment (georgegillams) [reply docs/src/pages/components/Text/AndroidText.mdx:19]: Actually maybe it's not. I can't decide. Anyway the paths can be simplified regardless 👍

**Review by shaundon — APPROVED** [review]

File: docs/src/pages/components/Text/AndroidText.mdx (hunk 1 above)
Comment (olliecurtis) [reply docs/src/pages/components/Text/AndroidText.mdx:19]: Hmm I should've done this already! Will check if I have missing commit

File: docs/src/pages/components/Text/NativeText.mdx (hunk 2 above)
Comment (olliecurtis) [reply docs/src/pages/components/Text/NativeText.mdx:17]: same as above

File: docs/src/pages/components/Text/AndroidText.mdx (hunk 1 above)
Comment (olliecurtis) [reply docs/src/pages/components/Text/AndroidText.mdx:19]: Looks like a merge from master messed it up a bit here! Commit incoming

**Review by shaundon — APPROVED** [review]

**Review by georgegillams — APPROVED** [review]

**Pull request merged**